contracts/CollateralTracker.sol

```diff
@@ -818,7 +818,7 @@ contract CollateralTracker is ERC20Minimal, Multicall {
 
     /// @notice Get the cost of exercising an option. Used during a forced exercise.
     /// @dev This one computes the cost of calling the forceExercise function on a position:
-    /// - The forceExercisor will have to *pay* the exercisee because their position will be closed "against their will"
```

Comment:
Please revert, exercise is not gramatically correct here. exercisee=user who is being force exercised